classification/train.py

```diff
@@ -153,7 +153,12 @@ def train(model: Any,
 
         if (i != 0 and i % (CONFIG.valid_freq) == 0):
             valid_start_time = datetime.datetime.now()
-            _, _, best_valid_cmap = valid(model, valid_loader, epoch + i / len(data_loader), best_valid_cmap, CONFIG)
+            _, _, best_valid_cmap = valid(model, 
+                                          valid_loader, 
+                                          epoch + i / len(data_loader), 
+                                          best_valid_cmap, 
+                                          CONFIG.valid_dataset_ratio,
```

Comment:
Just call CONFIG.valid_dataset_ratio in the function, save us an argument space

Reply:
Even better, check if epoch%1 = 0

Reply:
So we can do full validation at end

Reply:
remove cmap - use map instead

Reply:
all righty thanks sam